lab_5_scrapper/scrapper.py

```diff
@@ -13,70 +72,109 @@ def __init__(self, path_to_config: Path) -> None:
         """
         Initializes an instance of the Config class
         """
-        pass
+        self.path_to_config = path_to_config
+        self._validate_config_content()
+        config_dto = self._extract_config_content()
+        self._seed_urls = config_dto.seed_urls
+        self._num_articles = config_dto.total_articles
+        self._headers = config_dto.headers
+        self._encoding = config_dto.encoding
+        self._timeout = config_dto.timeout
+        self._should_verify_certificate = config_dto.should_verify_certificate
+        self._headless_mode = config_dto.headless_mode
 
     def _extract_config_content(self) -> ConfigDTO:
         """
         Returns config values
         """
-        pass
+        with open(self.path_to_config, encoding='utf-8') as f:
+            content = json.load(f)
+        return ConfigDTO(**content)
```

Comment:
what ** means here?

Reply:
it unpacks content of json file (dictionary)...so then we pass it as an argument to a function which has parameters with the same name as keywords in dictionary..for example, instead of writing ConfigDTO(headers=content['headers']) and etc, we can just unpack dictionary and all the parameters will be filled with values that are associated with specific keyword